src/jit/treelifeupdater.h

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#pragma once
+
+#include "compiler.h"
+
+//------------------------------------------------------------------------
+// TreeLifeUpdater: class that handles life variables changes according to a tree
```

Comment:
Although the existing code uses "life" in many places, most of them should be "live" IMO.
I think this would be clearer as "class that handles changes in variable liveness from a given tree"

Reply:
Fixed, thank you.